<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rotina de Teste</string>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #343436;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Botao_TesteTouch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

    background-color: rgb(87, 90, 97);
    color: rgb(255, 255, 255);
    border-radius:5
}

QPushButton:hover{

	background-color: rgb(113, 196, 223);
    color: rgb(34, 36, 43);
    border-radius:7
    
}
QPushButton:pressed{

	background-color: rgb(62, 101, 255);
    color: rgb(255, 255, 255);
    border-radius:10

}</string>
    </property>
    <property name="text">
     <string>Teste de Touch</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Botao_DiretorioTouch">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Box_Valordeloop_Touch">
    <property name="geometry">
     <rect>
      <x>531</x>
      <y>60</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Diretorio_EXE_Touch">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 8pt &quot;MS Shell Dlg 2&quot;;
border-radius:5</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Diretorio_EXE_Teclado">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 8pt &quot;MS Shell Dlg 2&quot;;
border-radius:5</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="Box_Valordeloop_Teclado">
    <property name="geometry">
     <rect>
      <x>531</x>
      <y>120</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Botao_TesteTeclado">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

    background-color: rgb(87, 90, 97);
    color: rgb(255, 255, 255);
    border-radius:5
}

QPushButton:hover{

	background-color: rgb(113, 196, 223);
    color: rgb(34, 36, 43);
    border-radius:7
    
}
QPushButton:pressed{

	background-color: rgb(62, 101, 255);
    color: rgb(255, 255, 255);
    border-radius:10

}</string>
    </property>
    <property name="text">
     <string>Teste de Teclado</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Botao_DiretorioTeclado">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="Texto_Loop">
    <property name="geometry">
     <rect>
      <x>547</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Loop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Botao_reset">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>14</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: white;
	background-color: #de0341;
	border-radius: 10;
}
QPushButton:hover{

	background-color: rgb(113, 196, 223);
    color: rgb(34, 36, 43);
    border-radius:7
    
}
QPushButton:pressed{
	border: 2px solid #de0341;
	background-color:white;
    color: #de0341;
    border-radius:10
}</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Diretorio_Salvamento">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 8pt &quot;MS Shell Dlg 2&quot;;
border-radius:5</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="Texto_Localdesalvamento">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Local de salvamento do relat√≥rio:</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Botao_DiretorioSalvamento">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>320</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
